testbeam_analysis/tools/plot_utils.py

```diff
@@ -1067,3 +1067,36 @@ def efficiency_plots(hit_hist, track_density, track_density_with_DUT_hit, effici
         output_fig.savefig()
     else:
         logging.warning('Cannot create efficiency plots, since all pixels are masked')
+
+
+def plot_track_angle(input_track_slopes, output_data, output_pdf, use_n_duts, n_bins):
+    direction = ['x', 'y', 'z']
+
+    with PdfPages(output_pdf) as output_fig:
+        for i in range(use_n_duts):  # counting number of progressed DUTs
+            for key in input_track_slopes:
+                for j in (0, 1):  # iterate over slopes
```

Comment:
Sorry what do you mean with this? j is the index for iterating over the slope directions.